tests/cggi/syntax.mlir

```diff
@@ -0,0 +1,21 @@
+// RUN: heir-opt -verify-diagnostics %s
+
+// This simply tests for syntax.
+#encoding = #lwe.bit_field_encoding<cleartext_start=30, cleartext_bitwidth=3>
+#poly = #polynomial.polynomial<1 + x**1024>
+#ring = #polynomial.ring<cmod=7917, ideal=#poly>
+!plaintext = !lwe.lwe_plaintext<encoding = #encoding>
+!ciphertext = !lwe.lwe_ciphertext<encoding = #encoding, ring = #ring>
+
+module {
+  func.func @test_syntax(%arg0 : !ciphertext) -> !ciphertext {
+    %0 = arith.constant 0 : i1
+    %1 = arith.constant 1 : i1
+    %2 = lwe.encode %0 { encoding = #encoding }: i1 to !plaintext
+    %3 = lwe.encode %1 { encoding = #encoding }: i1 to !plaintext
+    %4 = cggi.trivial_encrypt %2 { ring = #ring } : !plaintext to !ciphertext
+    %5 = cggi.trivial_encrypt %3 { ring = #ring } : !plaintext to !ciphertext
```

Comment:
note: i'm playing around here with the assembly format to see if it's possible to remove the `!plaintext`.